datastore/longhorn.go

```diff
@@ -482,7 +482,16 @@ func (s *DataStore) ListNodes() ([]*longhorn.Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	return nodeList, nil
+	nList := make([]*longhorn.Node, 0)
```

Comment:
Unless you have expected non-zero length, use `nlist := []*longhorn.Node{}` instead of `make`.